2020-01-Winter/5_graphs/Bipartite/Bipartite_Solution.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * ACM CTCI
+ * Graph Practices, https://leetcode.com/problems/is-graph-bipartite/
+ * @author Bill Zhao
+ * @date 02/14/2020
+ */
+
```

Comment:
It's important to have an executable here that includes a few tests. Could you add that?